plugins/hls-hlint-plugin/hls-hlint-plugin.cabal

```diff
@@ -53,13 +53,13 @@ library
     , transformers
     , unordered-containers
 
-  if ((!flag(ghc-lib) && impl(ghc >=8.10.1)) && impl(ghc <8.11.0))
+  if (!flag(ghc-lib) && impl(ghc >=8.10.1) && impl(ghc <9.0.0))
     build-depends: ghc ^>= 8.10
 
   else
     build-depends:
       , ghc
-      , ghc-lib            ^>= 8.10.2.20200916
+      , ghc-lib            ^>= 8.10.4.20210206
```

Comment:
Is it OK to not update cabal.project index?

Reply:
Yeah, that ghc-lib version was releases on 2021-02-06T16:51:55Z